doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -164,6 +164,44 @@ parameters. This profile should be used with caution and is meant for advanced
 users, who understand mclock and Ceph related configuration options.
 
 
+.. index:: mclock; shard config for HDD clusters
+
+OSD Shard Configuration For HDD Based Clusters With mClock
+==========================================================
+As described in :ref:`dmclock-qos-caveats`, the number of OSD shards employed
```

Comment:
I added a small paragraph explaining the meaning of OSD shard and associated worker threads to make it clearer.

Reply:
So is a shard in this context a fraction of the PG replicas on the given OSD?  Is it independent of PGs?
When I see "shard" I think erasure coding data / parity shards, so I think it would be good to be clear here if this does or doesn't mean the same thing.

Reply:
The shard terminology here is not related to erasure coding. It instead refers  to a unique OSD queue (managed by a scheduler - in this case mClock) which handles the various types of operations (client I/O, recovery, scrub etc.) on the OSD. The number of such queues on a given OSD is controlled by the _osd_op_num_shards_[ssd|hdd]_ config option.

If the above description is clear, I can add it to the section. Please let me know.

Reply:
Ah, thanks.  I think adding that would be a good idea.

Reply:
@anthonyeleven Done. Please verify.